src/custom/components/TransactionSettings/TransactionSetttingsMod.tsx

```diff
@@ -31,21 +31,24 @@ const FancyButton = styled.button`
   border: 1px solid ${({ theme }) => theme.bg3};
   outline: none;
   background: ${({ theme }) => theme.bg1};
-  :hover {
+
+  &:hover {
     border: 1px solid ${({ theme }) => theme.bg4};
   }
-  :focus {
+  &:focus {
     border: 1px solid ${({ theme }) => theme.primary1};
   }
 `
 
 const Option = styled(FancyButton)<{ active: boolean }>`
   margin-right: 8px;
-  :hover {
+  border: 0;
+  background-color: ${({ active, theme }) => active && theme.primary1};
+  color: ${({ theme }) => theme.text1};
+
+  &:hover {
```

Comment:
Ideally we don't touch the mod, this makes it hard for updates.

We can overlook it, but probably the way to go is to export `Option`, and to let `SlippageTabs` receive a `Option` component. Then the inde would hand in the custom component based in the original

Reply:
Updated now to not touch any Mod files directly. Please review.